### PR TITLE
Include ui in the flask app

### PR DESCRIPTION
**Pull request by TooAngel**
The angular ui is server from flask right now. Should be improved when properly deployed.

The js files are single flask routes. Could find a good way to keep it more generic.

### CODE REVIEW

**Comment by democratic-collaboration** [comment]
A new review, yeah.

    Votes: 75/78
    Coefficient: 0.961538461538
    Merging in 0 days 8 hours
    Age 0 days 0 hours

File: run.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+git pull
+. .env
+cd ui
+yarn
+node_modules/.bin/ng build
```
Comment (pst) [inline run.sh]: add `--prod` to `ng build` to produce bundles optimized for production.

File: src/server.py [hunk 2]
```diff
@@ -46,7 +45,28 @@ def token_getter():
 
 @app.route('/')
```
Comment (pst) [inline src/server.py]: I think you're misisng the styles.css here.

**Comment by democratic-collaboration** [comment]
A new review, yeah.

    Votes: 77/81
    Coefficient: 0.950617283951
    Merging in 0 days 9 hours
    Age 0 days 2 hours

**Comment by democratic-collaboration** [comment]
A new review, yeah.

    Votes: 76/82
    Coefficient: 0.926829268293
    Merging in 0 days 14 hours
    Age 0 days 2 hours

File: run.sh (hunk 1 above)
Comment (TooAngel) [inline run.sh]: k

Comment (TooAngel) [inline run.sh]: with `--prod` the static files get random name, which breaks the flask static file serving currently.

We have to fix that first, IMHO not part of this PR.

File: src/server.py (hunk 2 above)
Comment (TooAngel) [inline src/server.py]: Added

**Comment by democratic-collaboration** [comment]
A new review, yeah.

    Votes: 80/82
    Coefficient: 0.975609756098
    Merging in 0 days 5 hours
    Age 0 days 0 hours
